mobile/src/helpers/vendorMockData.js

```diff
@@ -0,0 +1,46 @@
+const vendorData = [
```

Comment:
we can delete things inside helpers/ folder since we're using /data folder now in HomeView.jsx